data/data_download.py

```diff
@@ -120,6 +120,7 @@ def load_and_execute_downloaders(
     hf_token: Optional[str] = None,
     trust_remote_code: bool = False
 ):
+    executed_downloaders = 0
```

Comment:
What's the purpose of this? It's never used and the error that some datasets weren't downloaded is printed always.